diceRoll/diceRollUnitTests.js

```diff
@@ -98,87 +93,87 @@ function testList() {
       app.toggleMenu = new Stub();
       app.userSaveButtonCheckDisplay = new Stub();
       app.run();
-      assert.compare(app.addRollBarListeners.info.called, 1, 'addRollBarListeners() not called');
-      assert.compare(app.addCalculatorListeners.info.called, 1, 'addCalculatorListeners() not called');
-      assert.compare(app.addUserSaveButtonListener.info.called, 1, 'addUserSaveButtonListener() not called');
-      assert.compare(app.simulateFirstVisit.info.called, 1, 'simulateFirstVisit() not called');
-      assert.compare(app.checkMemory.info.called, 1, 'checkMemory() not called');
-      assert.compare(app.toggleMenu.info.called, 1, 'toggleMenu() not called');
-      assert.compare(app.userSaveButtonCheckDisplay.info.called, 1, 'userSaveButtonCheckDisplay() not called');
+      assert.compare(app.addRollBarListeners.info.calls.length, 1, 'addRollBarListeners() not called');
+      assert.compare(app.addCalculatorListeners.info.calls.length, 1, 'addCalculatorListeners() not called');
+      assert.compare(app.addUserSaveButtonListener.info.calls.length, 1, 'addUserSaveButtonListener() not called');
+      assert.compare(app.simulateFirstVisit.info.calls.length, 1, 'simulateFirstVisit() not called');
+      assert.compare(app.checkMemory.info.calls.length, 1, 'checkMemory() not called');
+      assert.compare(app.toggleMenu.info.calls.length, 1, 'toggleMenu() not called');
+      assert.compare(app.userSaveButtonCheckDisplay.info.calls.length, 1, 'userSaveButtonCheckDisplay() not called');
       return true;
     }),
     new UnitTest('addCalculatorListeners()', function(app, test) {
       app.addNumberKeyListeners = new Stub();
       app.addDiceKeyListeners = new Stub();
       app.addOperatorKeyListeners = new Stub();
       app.addCalculatorListeners();
-      assert.compare(app.addNumberKeyListeners.info.called, 1, 'addNumberKeyListeners() not called');
-      assert.compare(app.addDiceKeyListeners.info.called, 1, 'addDiceKeyListeners() not called');
-      assert.compare(app.addOperatorKeyListeners.info.called, 1, 'addOperatorKeyListeners() not called');
+      assert.compare(app.addNumberKeyListeners.info.calls.length, 1, 'addNumberKeyListeners() not called');
+      assert.compare(app.addDiceKeyListeners.info.calls.length, 1, 'addDiceKeyListeners() not called');
+      assert.compare(app.addOperatorKeyListeners.info.calls.length, 1, 'addOperatorKeyListeners() not called');
       return true;
     }),
     new UnitTest('addNumberKeyListeners()', function(app, test) {
       app.context.attach = new Stub();
```

Comment:
Name: "addNumberKeyListeners() attaches event listeners to each of the Number buttons on the calculator by calling bind on the keypadPress method 10 times"